package.json

```diff
@@ -14,7 +14,12 @@
       "import": "./index.mjs",
       "require": "./index.js"
     },
-    "./*": "./*"
+    "./browser": "./browser/index.js",
```

Comment:
This is needed since the new `"./*"` definition would make importing of `"typeorm/browser"` look for `node_modules/typeorm/browser.js` instead of `node_modules/typeorm/browser/index.js`, so this property makes sure importing `"typeorm/browser"` continues to work properly